app/src/main/java/com/movies/common/extensions/ViewExtension.kt

```diff
@@ -25,6 +27,11 @@ fun View.changeBackgroundColor(@ColorRes colorRes: Int) {
     backgroundTintList = ContextCompat.getColorStateList(context, colorRes)
 }
 
+fun Button.setCategoryStyle(style: CategoryStyle) {
```

Comment:
You only use this extension in Adapter and why don't you write it inside it?
Also, it is not good idea to have specific thing (`CategoryStyle`) in global extensions.

Reply:
it can be used on other buttons as well if needed. Therefore, I changed the name into a general one